scrape_rec/spiders/base_realestate.py

```diff
@@ -56,8 +56,13 @@ def process_link(self, response):
 
         ad_date = None
         if self.date_xpath:
-            ad_date = response.xpath(self.date_xpath).extract_first().strip()
-            ad_date = self.process_ad_date(ad_date)
+            try:
+                ad_date = response.xpath(self.date_xpath).extract_first().strip()
+                ad_date = self.process_ad_date(ad_date)
+            except:
+                self.logger.error('Problem with date xpath, falling back to current date')
+                ad_date = None
+
```

Comment:
This could all be removed with some itemloaders:

`
loader.add_xpath('some_field',  '//some/xpath')
`
The strip is in the background and you don't have to worrry about it failing